<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUp</class>
 <widget class="QDialog" name="SignUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>341</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="verticalSpacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="showLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="setBtn">
      <property name="text">
       <string>设置</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="nameLabel">
      <property name="text">
       <string>用户名:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QLineEdit" name="nameLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="passwordLabel">
      <property name="text">
       <string>密码:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QLineEdit" name="passwordLineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="checkPasswordLabel">
      <property name="text">
       <string>再次输入密码:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QLineEdit" name="checkPasswordLineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="pwQuestionLabel">
      <property name="text">
       <string>密保问题:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="2">
     <widget class="QComboBox" name="pwQuestionComboBox">
      <item>
       <property name="text">
        <string>你机网实验的成绩是多少?</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="pwAnswerLabel">
      <property name="text">
       <string>密保答案:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="1" colspan="2">
     <widget class="QLineEdit" name="pwAnswerLineEdit"/>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QPushButton" name="signUpBtn">
      <property name="text">
       <string>注册</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QPushButton" name="exitBtn">
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
